### PR TITLE
DOCS-872: Update condition key list

### LINKED ISSUE #872: Can't define policy with ExistingObjectTag on PUT object
## Expected Behavior

Being able to apply rights as described in the documentation: https://min.io/docs/minio/linux/administration/identity-access-management/policy-based-access-control.html#supported-s3-policy-condition-keys with PutObject / PutObjectTagging 

## Current Behavior
Trying to submit the following policy :
`
{
    "Version": "2012-10-17",
    "Statement": [
      {
        "Effect": "Allow",
        "Action": ["s3:GetBucketLocation", "s3:HeadBucket", "s3:ListBucket"],
        "Resource": "arn:aws:s3:::test"
      },
      {
        "Effect": "Allow",
        "Action": ["s3:GetObject"],
        "Resource": "arn:aws:s3:::test/*"
      },
      {
        "Effect": "Deny",
        "Action": ["s3:PutObjectTagging", "s3:PutObject"],
        "Resource": "arn:aws:s3:::test/*",
        "Condition": {
            "StringEquals":
              {"s3:ExistingObjectTag/sealed": "on"}
          }
      }
  ]
}
`

Fails with : 

> There was an error creating a Policy: unsupported condition keys '[s3:ExistingObjectTag/sealed]' used for action 's3:PutObject'. Please check Policy syntax.

## Possible Solution
Update the documentation or code.

## Steps to Reproduce (for bugs)

1. Open the mini console
2. Try to submit the policy in the current behavior description.

## Context
Trying to use object tagging to avoid put on message (versioning mgt is too heavy/complex for our use case).

## Regression
Don't know as my previous version of minio does not support it.

## Your Environment

* mini docker image RELEASE.2023-05-04T21-44-30Z
* single node server.

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #872 

Synchronizes the current list of policy actions to conditions

Used https://github.com/minio/pkg/blob/main/iam/policy/action.go and https://github.com/minio/pkg/blob/main/iam/policy/condition/keyname.go to refresh the list.

Also did some slight formatting cleanups, joined a bunch of lines, fixed a few minor linker errors.

Staged: http://192.241.195.202:9000/staging/DOCS-872/linux/administration/identity-access-management/policy-based-access-control.html#supported-s3-policy-actions

### CODE REVIEW

File: source/administration/identity-access-management/policy-based-access-control.rst [hunk 1]
```diff
@@ -815,8 +896,7 @@ MinIO supports the following conditions for use with defining policies for
 - ``aws:CurrentTime``
 - ``aws:EpochTime``
 
-For complete information on any listed condition key, see the :iam-docs:`IAM
-Condition Element Documentation <reference_policies_elements_condition.html>`
+For complete information on any listed condition key, see the
```
Comment (feorlen) [inline source/administration/identity-access-management/policy-based-access-control.rst:899]: Seems the second part of this sentence was eaten by a grue.

**Review by feorlen — CHANGES_REQUESTED** [review]
One to-be-reformatted line got accidentally nuked. Rest looks good.

File: source/administration/identity-access-management/policy-based-access-control.rst (hunk 1 above)
Comment (feorlen) [reply source/administration/identity-access-management/policy-based-access-control.rst:899]: Duh, I can just suggest it back in. Presumably it should look like this:
```suggestion
For complete information on any listed condition key, see the :iam-docs:`IAM Condition Element Documentation <reference_policies_elements_condition.html>`.
```

**Comment by ravindk89** [comment]
@feorlen suggestion committed, thanks!

**Review by feorlen — APPROVED** [review]
👍🏻

**Review by djwfyi — APPROVED** [review]
LGTM

**Comment by ravindk89** [comment]
@donatello feel free to issue corrections in this PR and I will merge them up later.

**Pull request merged**